src/schema/index.js

```diff
@@ -92,26 +97,25 @@ input VoteFilter {
 
 type Mutation {
   createTopic(
+    version: Int!
```

Comment:
version can be optional when create.

Reply:
removed. doesn't make sense to have it for the unconfirmed items.

Reply:
do I need to make this optional in the `typeDefs` to allow for this?

Reply:
lets leave mutation as it is because not really use it